test/integration/auth_test.go

```diff
@@ -393,6 +393,7 @@ func TestAuthModeAlwaysAllow(t *testing.T) {
 		EnableUISupport:       false,
 		EnableIndex:           true,
 		APIPrefix:             "/api",
+		EnableV1Beta3:         true,
```

Comment:
Because we still run integration tests for both v1 and v1beta3, to ensure that it will work if ppl turn it on using the right flags.
In the next release, we will delete all support for the API and stop running the tests.